appservice/registration.go

```diff
@@ -21,9 +21,10 @@ type Registration struct {
 	AppToken        string     `yaml:"as_token"`
 	ServerToken     string     `yaml:"hs_token"`
 	SenderLocalpart string     `yaml:"sender_localpart"`
-	RateLimited     bool       `yaml:"rate_limited"`
+	RateLimited     bool       `yaml:"rate_limited,omitempty"`
```

Comment:
Won't this omit `false`? It probably needs to be a pointer to allow not setting the value

Reply:
Oh yes it would, good point.

Reply:
Fixed in https://github.com/tulir/mautrix-go/pull/20/commits/ae55ddbaec64041b0d663c9d9ff3348603e0d066